lib/moon_web/pages/components/menu_item_page.ex

```diff
@@ -0,0 +1,772 @@
+              <MenuItem.Title>Your value</MenuItem.Title>
+              <span class="text-moon-12 text-trunks-100">Meta</span>
+            </MenuItem>
+          </div>
+        </:example>
+        <:code>{menu_item_4_code()}</:code>
+      </ExampleAndCode>
+
+      <ExampleAndCode title="Checkbox" id="menu_5">
+        <:example>
+          <div class="w-56">
+            <MenuItem role="checkbox" title="Your value" on_click="on_select1" is_selected={@selected1} />
+          </div>
+          <div class="w-56">
+            <MenuItem role="checkbox" on_click="on_select0" is_selected={@selected0}>
+              <MenuItem.Title>Your value</MenuItem.Title>
+              <MenuItem.Checkbox />
+            </MenuItem>
+          </div>
+        </:example>
+        <:code>{menu_item_5_code()}</:code>
+      </ExampleAndCode>
+
+      <ExampleAndCode title="Radio" id="menu_9">
+        <:example>
+          <div class="w-56">
+            <MenuItem role="radio" on_click="on_select3" is_selected={@selected3}>
+              <MenuItem.Radio />
+              <MenuItem.Title>Your value</MenuItem.Title>
+            </MenuItem>
+          </div>
+          <div class="w-56">
+            <MenuItem role="radio" on_click="on_select4" is_selected={@selected4}>
+              <MenuItem.Title>Your value</MenuItem.Title>
+              <MenuItem.Radio />
+            </MenuItem>
+          </div>
+        </:example>
+        <:code>{menu_item_9_code()}</:code>
+      </ExampleAndCode>
+
+      <ExampleAndCode title="MultiTitle" id="menu_6">
+        <:example>
+          <div class="w-56">
+            <MenuItem>
+              <span class="px-2">
+                <ControlsChevronRight class="text-[1.5rem]" />
+              </span>
+              <MenuItem.MultiTitle title="Your value">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry
+              </MenuItem.MultiTitle>
+              <span class="text-moon-12 text-trunks-100">Meta</span>
+            </MenuItem>
+          </div>
+          <div class="w-56">
+            <MenuItem role="checkbox" on_click="on_select2" is_selected={@selected2}>
+              <span class="px-2">
+                <SportBadminton class="text-[1.5rem]" />
+              </span>
+              <MenuItem.MultiTitle title="Your value">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry
+              </MenuItem.MultiTitle>
+              <MenuItem.Checkbox />
+            </MenuItem>
+          </div>
+        </:example>
+        <:code>{menu_item_6_code()}</:code>
+      </ExampleAndCode>
+
+      <ExampleAndCode title="Multi Line Items" id="menu_7">
+        <:example>
+          <div class="w-94 bg-gohan-100 flex flex-col gap-3 rounded-moon-s-lg p-6">
+            <MenuItem is_active>
+              <span class="bg-piccolo-100/20 rounded-lg w-10 h-10 flex justify-center items-center">
```

Comment:
I am afraid, that if in our examples there is a lot of CSS, then developers will use a lot of CSS. Maybe some component would help with default looks?